data/alpha/field/hostname.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: hostname
+phase: alpha
+register: government-service
```

Comment:
s/register: government-service/register: ''/

As this is not a primary key field, we don't add a register value on this line.